apps/src/templates/manageStudents/ManageStudentsLoginInfo.jsx

```diff
@@ -99,13 +101,16 @@ class ManageStudentsLoginInfo extends Component {
         />
         <h2>{i18n.privacyHeading()}</h2>
         <p id="uitest-privacy-text">{i18n.privacyDocExplanation()}</p>
-        <Button
-          __useDeprecatedTag
-          color="white"
+        <DownloadParentLetter
+          sectionId={this.props.sectionId}
+          buttonLocation={ParentLetterEntryPoint.BELOW_TABLE}
+        />
+        <br />
+        <SafeMarkdown
           id="uitest-privacy-link"
-          target="_blank"
-          href={pegasus('/privacy/student-privacy')}
-          text={i18n.privacyButton()}
+          markdown={i18n.privacyLinkToPolicy({
+            privacyPolicyLink: pegasus('/privacy/student-privacy')
+          })}
         />
```

Comment:
@Hamms is there a way to log clicks on a SafeMarkdown link? Or maybe @breville is this an example where I should us Google Analytics rather than Firehose?

Reply:
Not that I'm aware of; you could theoretically attach an event listener to the link after rendering, but it's kinda messy.

To be clear, are you interested in tracking clicks on this link or visits to the student-privacy page?

Reply:
I'm interested in tracking clicks on this link. There are a few ways you can get to the student-privacy page, and we want to know how much usage this particular link gets.

Reply:
In Google Analytics you can look at page views for a specific page and then look at a "navigation summary" which shows the on-site pages that users were on beforehand.

We've also used Google Tag Manager which can trigger Google Analytics events before doing the subsequent navigation, but that would mainly be useful if we had more than one link to the same destination on the same page, which I don't think is the case here.